codegen/tests/unit/responses/iam-list-access-keys.xml

```diff
@@ -0,0 +1,20 @@
+<ListAccessKeysResponse xmlns="https://iam.amazonaws.com/doc/2010-05-08/">
```

Comment:
This file is in the codegen botocore's dir - can we refer to that one?  I assume this file is left over from testing.